instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -584,20 +636,31 @@ def __call__(self, environ, start_response):
         try:
             with trace.use_span(span):
                 start_response = self._create_start_response(
-                    span, start_response, response_hook, duration_attrs
+                    span,
+                    start_response,
+                    response_hook,
+                    req_attrs,
+                    self._sem_conv_opt_in_mode,
                 )
                 iterable = self.wsgi(environ, start_response)
                 return _end_span_after_iterating(iterable, span, token)
         except Exception as ex:
             if span.is_recording():
+                if _report_new(self._sem_conv_opt_in_mode):
+                    span.set_attribute(_SPAN_ATTRIBUTES_ERROR_TYPE, type(ex).__qualname__ )
```

Comment:
I think we should also add it to the duration_attrs_new (regardless of `span.is_recording`)